docs/content/environment_tests.md

```diff
@@ -61,21 +61,23 @@ The second optional argument, `test_kept_state` allows the user to disable the s
 
 The max cycles test tests that the `max_cycles` environment argument exists and the resulting environment actually runs for the correct number of cycles. If your environment does not take a `max_cycles` argument, you should not run this test. The reason this test exists is that many off-by-one errors are possible when implementing `max_cycles`. An example test usage looks like:
 
-``` python
+``` python notest
 from pettingzoo.test import max_cycles_test
 from pettingzoo.butterfly import pistonball_v6
 env = pistonball_v6.env()
+#TODO AttributeError: 'OrderEnforcingWrapper' object has no attribute 'parallel_env'
 max_cycles_test(env)
 ```
 
 ## Render Test
 
 The render test checks that rendering 1) does not crash and 2) produces output of the correct type when given a mode (only supports `'human'`, `'ansi'`, and `'rgb_array'` modes).
 
-``` python
+``` python notest
 from pettingzoo.test import render_test
 from pettingzoo.butterfly import pistonball_v6
 env = pistonball_v6.env()
+#TODO TypeError: 'OrderEnforcingWrapper' object is not callable
```

Comment:
Also sounds like a Pistonball issue

Reply:
Alright here it looks like the render_test() was expecting to be passed the function to initialize the environment (`pistonball_v6.env`) instead of `pistonball_v6.env()`. I added a type annotation to `render_test` to avoid confusion in the future

Reply:
Thanks this is useful to have